src/main/java/org/launchcode/water_garden_tour/controllers/HomeController.java

```diff
@@ -1,20 +1,24 @@
 package org.launchcode.water_garden_tour.controllers;
 
+import org.launchcode.water_garden_tour.models.data.FeatureRepository;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
-import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.GetMapping;
 
 @Controller
 public class HomeController {
 
-    @RequestMapping("")
+    @Autowired
+    private FeatureRepository featureRepository;
+
+    @GetMapping("")
```

Comment:
why did we change RequestMapping to GetMapping here?

Reply:
I thought it was the more correct annotation since it specifies the type of request. It was working fine before so I can definitely leave it as RequestMapping.

Reply:
sounds good to me thanks for the explanation!